app-insights.tf

```diff
@@ -33,3 +33,11 @@ output "connection_string" {
 output "app_id" {
   value = azurerm_application_insights.this.app_id
 }
+
+output "appinsight_name" {
+  value = azurerm_application_insights.this.name
+}
+
+output "appinsight_location" {
```

Comment:
this seems a bit silly it's a direct input?

Reply:
Yeah - direct input is "UK South", and if it reads from the resource its `uksouth`  plan shows this changes so trying to get around that.
https://build.hmcts.net/blue/organizations/jenkins/HMCTS_j_to_z%2Fsscs-shared-infrastructure/detail/PR-167/5/pipeline/